app/routes/search-index.js

```diff
@@ -32,7 +32,7 @@ module.exports = [{
       if (!cachedPage?.success) {
         await auditSearch(request.params.terms, request.query, user)
 
-        const results = await search(user, request.params.type, request.params.terms, !!request.query.fuzzy, !!request.query.national)
+        const results = await search(request, user, request.params.type, request.params.terms, !!request.query.fuzzy, !!request.query.national)
```

Comment:
not a big one and happy to be proved wrong, but if you are passing request, are the other arguments still needed?  They may be?